test/dotnet.Tests/GivenThatTheUserIsRunningDoNetForTheFirstTime.cs

```diff
@@ -28,15 +29,24 @@ static GivenThatTheUserIsRunningDotNetForTheFirstTime()
             command.Environment["NUGET_PACKAGES"] = testNugetCache;
             command.Environment["DOTNET_SKIP_FIRST_TIME_EXPERIENCE"] = "";
 
-            _firstDotnetUseCommandResult = command.ExecuteWithCapturedOutput("new");
+            _firstDotnetInfoUseCommandResult = command.ExecuteWithCapturedOutput("--info");
+            _firstDotnetNewUseCommandResult = command.ExecuteWithCapturedOutput("new");
 
             _nugetCacheFolder = new DirectoryInfo(testNugetCache);
         }        
 
         [Fact]
         public void Using_dotnet_for_the_first_time_succeeds()
         {
-            _firstDotnetUseCommandResult.Should().Pass();
+            _firstDotnetNewUseCommandResult.Should().Pass();
+        }
+
+        [Fact]
+        public void Using_dotnet_for_the_first_time_with_non_verbs_doesnt_print_eula()
+        {
+            const string firstTimeUseWelcomeMessage = @".NET Command Line Tools";
```

Comment:
The naming of this variable is somewhat confusing, when I looked at the assertion below I was confused for a bit on why `dotnet --info` would be showing a first time message. Could we rename it to something like `nonVerbHeaderMessage` or something that implies that it's not part of the first time experience.